README.md

```diff
@@ -1,6 +1,11 @@
 # Stargate
+An open source data gateway.
 
-Discord bot using Stargate
+Stargate is a data gateway deployed between client applications and a database. It's built with extensibility as a first-class citizen and makes it easy to use a database for any application workload by adding plugin support for new APIs, data types, and access methods.
```

Comment:
If this is from their website, I suggest quoting... 

```suggestion
> Stargate is a data gateway deployed between client applications and a database. It's built with extensibility as a first-class citizen and makes it easy to use a database for any application workload by adding plugin support for new APIs, data types, and access methods.
```

This will look like ...
> Stargate is a data gateway deployed between client applications and a database. It's built with extensibility as a first-class citizen and makes it easy to use a database for any application workload by adding plugin support for new APIs, data types, and access methods.

Reply:
this is correct, but looks less beautiful. Can we incorporate a third method?